### PR TITLE
integrate submission to sequencer

### PR DESCRIPTION
**Pull request by noot**
similar to how txs were submitted directly to metro before, this PR submits txs directly to the rust sequencer.

### CODE REVIEW

File: internal/ethapi/sequencer.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+const rollupChainID = "ethereum"
+
+func sendTransactionToSequencer(ctx context.Context, txBytes []byte) error {
+	signer, err := client.GenerateSigner()
+	if err != nil {
+		return err
+	}
+
+	// default tendermint RPC endpoint
+	c, err := client.NewClient("http://localhost:26657")
```
Comment (joroshiba) [inline internal/ethapi/sequencer.go]: Same here.

File: internal/ethapi/sequencer.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+import (
+	"context"
+	"fmt"
+
+	client "github.com/astriaorg/go-sequencer-client"
+	sqproto "github.com/astriaorg/go-sequencer-client/proto"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+const rollupChainID = "ethereum"
```
Comment (joroshiba) [inline internal/ethapi/sequencer.go]: Can we make configurable on startup?

File: internal/ethapi/sequencer.go (hunk 1 above)
Comment (noot) [reply internal/ethapi/sequencer.go]: done, I just used env vars, let me know if that works

File: internal/ethapi/sequencer.go [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	var (
+		chainID, tendermintRPCEndpoint string
+	)
+
+	envChainId := os.Getenv("CHAIN_ID")
+	if envChainId == "" {
+		chainID = defaultChainID
+	}
+
+	envTendermintRPCEndpoint := os.Getenv("TENDERMINT_RPC_ENDPOINT")
```
Comment (joroshiba) [inline internal/ethapi/sequencer.go]: nit: cometbft

**Review by joroshiba — APPROVED** [review]

**Pull request merged**